osquery/tables/system/posix/magic.cpp

```diff
@@ -19,61 +22,37 @@ namespace tables {
 
 namespace {
 
-constexpr std::array<const char*, 3> kMagicFiles = {{
-    "/usr/share/misc/magic",
```

Comment:
Is the removal of this intentional?

Reply:
Yes, per documentation a folder is only relevant if you plan to use the compile APIs.